bin/MadGraph5_aMCatNLO/cards/production/13TeV/VLferm_EWcouplings_4321/VLferm_EWcouplings_4321_m900GeV_EE_to_4b_customizecards.dat

```diff
@@ -0,0 +1,7 @@
+../../../../cards/production/13TeV/VLferm_EWcouplings_4321/param_card.dat
```

Comment:
please don't include relative paths that might not work. the default settings of the model will anyway be loaded and the customize card is just to adjust the rest.

Reply:
Okay, I've removed it and written out all of the differences explicitly.